v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FExtractions.scala

```diff
@@ -0,0 +1,256 @@
+package org.broadinstitute.monster.etl.v2f
+
+import java.io.InputStreamReader
+import java.nio.channels.Channels
+
+import better.files._
+import com.github.tototoshi.csv.{CSVFormat, CSVReader, TSVFormat}
+import com.spotify.scio.ScioContext
+import com.spotify.scio.values.SCollection
+import io.circe.{Json, JsonObject}
+import org.apache.beam.sdk.io.FileIO
+import org.apache.beam.sdk.io.FileIO.ReadableFile
+
+import scala.util.matching.Regex
+import util.Try
+
+/**
+  * Ingest step converting and tranforming tsv's from v2f.
+  *
+  */
+object V2FExtractions {
```

Comment:
This class has methods that fall under both `E` and `T`... it'd probably be clearest to just call it `V2FUtils`